templates/components/form/rulematchedlogs.html.twig

```diff
@@ -0,0 +1,106 @@
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% block more_fields %}
+    {{ printerAjaxPager|raw }}
+    <table class="tab_cadre_fixe" cellpadding="1">
+        <tr>
+            <th colspan="5"> {{ __('Rule import logs') }} </th>
+        </tr>
+        <tr>
+            <th> {{ _n('Date', 'Dates', 1) }}</th>
+            <th>{{ __('Rule name') }}</th>
+            {% if pagetype == 'Agent' %}
+                <th>{{ __('Item Type') }}</th>
+                <th>{{ _n('Item', 'Items', 1) }}</th>
+                <th>{{ __('Module') }}</th>
+            {% else %}
+                <th>{{ agenttypename }}</th>
+                <th>{{ __('Module') }}</th>
+                <th>{{ __('Criteria') }}</th>
+            {% endif %}
+        </tr>
+        {% for data in datas %}
+            <tr>
+                <td>{{ data['date'] }}</td>
+                <td>{{ data['rulelink']|raw }}</td>
+                {% if pagetype == 'Agent' %}
+                    <td>{{ data['itemtype'] }}</td>
+                    <td>{{ data['itemlink']|raw }}</td>
+                    <td>{{ data['modulename'] }}</td>
+                {% else %}
+                    <td>{{ data['agentlink']|raw }}</td>
+                    <td>{{ data['modulename'] }}</td>
+                    <td>
+                        {% if data['criteria'] is not empty %}
+                            <span class="toggle">
+                                <button class="btn">
+                                    <i class="ti ti-eye"></i>
+                                </button>
+                            </span>
+                            <ul class="nested list-group list-group-flush card" style="display: none;">
+                                <div class="list-group-item ">
+                                    {{ data['criteria']|raw }}
+                                </div>
+                            </ul>
+                        {% else %}
+                            {{ __('No criteria') }}
+                        {% endif %}
+                    </td>
+                {% endif %}
+            </tr>
+        {% endfor %}
+    </table>
+    {{ printerAjaxPager|raw }}
+{% endblock %}
+
+<script>
+  // Get all toggle buttons
+  var toggleButtons = document.querySelectorAll(".toggle");
+
+  // Add click event listeners to toggle buttons
+  toggleButtons.forEach(function(button) {
+    button.addEventListener("click", function() {
+      // Get the nested list
+      var nestedList = button.nextElementSibling;
+      // Toggle the display of the nested list
+      if (nestedList.style.display === "none") {
+        nestedList.style.display = "block";
+        button.innerHTML = '<button class="btn"><i class="ti ti-eye-off"></i></div>';
+      } else {
+        nestedList.style.display = "none";
+        button.innerHTML = '<button class="btn"><i class="ti ti-eye"></i></button>';
+      }
+    });
+  });
+</script>
```

Comment:
this should by handle with boostrap (only) 
 with https://getbootstrap.com/docs/4.0/components/collapse/#accordion-example

But we need to bump bbostrap version to latest
https://getbootstrap.com/